libs/blocks/merch/three-in-one.js

```diff
@@ -20,6 +20,46 @@ export const LANA_OPTIONS = {
   tags: 'three-in-one',
 };
 
+export const reloadIframe = ({ iframe, theme, msgWrapper, handleTimeoutError }) => {
+  if (!msgWrapper || !iframe || !theme || !handleTimeoutError) return;
+  msgWrapper.remove();
+  iframe.setAttribute('data-wasreloaded', true);
+  iframe.style.display = 'block';
+  // eslint-disable-next-line no-self-assign
+  iframe.src = iframe.src;
+  iframe.classList.add('loading');
+  theme.style.display = 'block';
+  setTimeout(handleTimeoutError, 15000);
+};
+
+export const showErrorMsg = ({ iframe, miloIframe, showBtn, theme, handleTimeoutError }) => {
+  theme.style.display = 'none';
+  iframe.style.display = 'none';
+  const iconAndText = `
+  <div class="icon-and-text">
+    <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24">
+      <defs>
+        <style>
+          .fill {
+            fill: #fff;
+          }
+        </style>
+      </defs>
+      <title>S Alert 24 N</title>
+      <rect id="Canvas" fill="#ff13dc" opacity="0" width="24" height="24" />
+      <path class="fill"
+        d="M11.418,1.72L0.267,21.675c-0.134,0.238,0.033,0.535,0.581,0.535h22.305c0.547,0,0.714-0.297,0.581-0.535L12.582,1.72c-0.134-0.238-1.031-0.238-1.164,0ZM13.333,19.669c0,0.333-0.333,0.333-0.333,0.333h-2c0,0-0.333,0-0.333-0.333v-2c0-0.333,0.333-0.333,0.333-0.333h2c0,0,0.333,0,0.333,0.333Zm0-4c0,0.333-0.333,0.333-0.333,0.333h-2c0,0-0.333,0-0.333-0.333v-8c0-0.333,0.333-0.333,0.333-0.333h2c0,0,0.333,0,0.333,0.333Z" />
+    </svg>
```

Comment:
@mirafedas I think we can replace this SVG content here with just one line in CSS. 
Download the SVG file from Spectrum page with icons (I believe you did that)
Optimize the file, here for example svgoptimizer.com to eliminate unnecessary tags like `title`, `rect`, etc.
Convert the optimized SVG to CSS here for example https://www.svgbackgrounds.com/tools/svg-to-css/
(Maybe there are better ways to do it, I'm just giving one example)
Use it in CSS as 
```
.icon-and-text:before {
   background-image: url('data:image/svg+xml,<svg ... </svg>');
}
```
You can check https://github.com/adobecom/milo/blob/stage/libs/features/mas/src/global.css.js#L288

Reply:
Discuss in DM, no change needed, resolving